src/CarbonAware.WebApi/src/Controllers/SciScoreController.cs

```diff
@@ -74,6 +74,37 @@ public async Task<IActionResult> GetCarbonIntensityAsync(SciScoreInput input)
         }
     }
 
+    /// <summary>
+    /// Given an array of time intervals (with one given location), retrieve the actual carbon intensity that would have occurred 
+    /// if the job was run at that time
+    /// </summary>
+    /// <remarks>
+    /// This endpoint takes a batch of time intervals for actual carbon emissions data (for one given location), fetches them, and calculates the actual
+    /// marginal carbon intensity values for the given list of time intervals to return the carbon intensity used during each interval.
+    ///
+    /// This endpoint is useful for figuring out how much carbon usage there would have been if the job was run at a specific date and time
+    /// </remarks>
+    /// <param name="requestActuals"> Array of requested actual values.</param>
+    /// <returns>An array of actual carbon intensity values during the given time interval.</returns>
+    /// <response code="200">Returns an array of responses which each contain a time interval and calculated carbon intensity values </response>
+    /// <response code="400">Returned if any of the requested items are invalid</response>
+    /// <response code="500">Internal server error</response>
+    /// <response code="501">Returned if the underlying data source does not support getting the data needed to calculate sci-scores</response>
+    [Produces("application/json")]
+    [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(IEnumerable<SciScore>))]
+    [ProducesResponseType(StatusCodes.Status400BadRequest, Type = typeof(ValidationProblemDetails))]
+    [ProducesResponseType(StatusCodes.Status500InternalServerError, Type = typeof(ValidationProblemDetails))]
+    [ProducesResponseType(StatusCodes.Status501NotImplemented, Type = typeof(ValidationProblemDetails))]
+    [HttpPost("marginal-carbon-intensity/batch")]
```

Comment:
I have two big-ish comments here:

1. If we are going to make this a batch version of `marginal-carbon-intensity` I think it is cleaner have the payload be an array of `SciScoreInput` objects.  Is there a particular reason why we introduce a new object here?

2. I am increasingly of the opinion that the `/sci-scores/marginal-carbon-intensity` endpoint is not the right place for this type of data and should actually be another `/emissions` endpoint to calculate the average carbon intensity over a period.  This could be an opportunity to address the fact that both `marginal-carbon-intensity` endpoints are more fundamentally aligned with the `emissions` data.  Keeping them in `sci-scores` has been a source of some confusion and it will likely become more confusing when `sci-scores` is full-featured to include energy and embodied emissions details.